deeprankcore/trainer.py

```diff
@@ -538,6 +539,8 @@ def train( # pylint: disable=too-many-arguments, too-many-branches, too-many-loc
                         Default: None.
             earlystop_maxgap (float, optional): Training ends if the difference between validation and training loss exceeds this value.
                         Default: None. 
+            min_epoch (float, optional): Minimum epoch to be reached before looking at maxgap.
+                        Default: 10
```

Comment:
Do we only want to use a min_epoch for max_gap, or for all early stopping? Or do we want to separate min_epoch arguments for each of those?
Also, i think we should be explicit that this argument is only used if max_gap (and/or patience) is not None.
Also, I think for all three earlystopping arguments, we should explicitly say in the docstring what happens when set to None.

```suggestion
            min_epoch (float, optional): Minimum epoch to be reached before looking at maxgap.
                        If maxgap is set to None, this parameter is ignored.
                        Default: 10
```